docs/usage.md

```diff
@@ -13,6 +13,7 @@ For more specific integration options, please check out the [integration documen
 - [Generate models from AsyncAPI documents](#generate-models-from-asyncapi-documents)
 - [Generate models from JSON Schema documents](#generate-models-from-json-schema-documents)
 - [Generate models from Swagger 2.0 documents](#generate-models-from-swagger-20-documents)
+- [Generate models from DataModel](#generate-models-from-datamodel)
```

Comment:
The name `DataModel` here seems to refer to a solution or a language. I feel it can confuse the reader. How do you feel about changing it to something like "Generate models from any other input type" or similar?

Reply:
I like it!

Reply:
That or "Generate models from our `DataModel`", to qualify it

Reply:
Definitely also an alternative 🤔 

Guess it depends on who reads the document. Those who are looking for how to generate models using the DataModel, or those who wish to generate models from anything else and don't know what `DataModel` means.

Reply:
> those who wish to generate models from anything else and would not know what `DataModel` means.

This would be, IMHO, the general use case. In fact, I'm not so sure those are the "data models" people may know about. In my case, the data model concept is abstract, and I don't see a "code" representation of it but just a diagram. I might be wrong, but I'm representing as well a user of this tool so I hope my insights are useful as well and perhaps also shared with other users that may come to this doc.

Reply:
Please see https://github.com/asyncapi/modelina/pull/530#issuecomment-1042000476